client/src/app/router.coffee

```diff
@@ -0,0 +1,11 @@
+Marionette = require('backbone.marionette')
+Controller = require('./controller')
+
+module.exports = Marionette.AppRouter.extend(
+  controller: new Controller()
+  appRoutes:
+    'new_site': 'new_site'
+    'list_sites': 'list_sites'
```

Comment:
I actually thought initially that would create naming conflicts, but either way, I just updated it.